src/update/UpdateManager.h

```diff
@@ -36,7 +36,7 @@ class UpdateManager final : public wxEvtHandler
     static UpdateManager& GetInstance();
     static void Start();
 
-    void GetUpdates();
+    void GetUpdates(bool ignoreNetworkErrors);
```

Comment:
An argument is made, but there is only one call so far, and always passing true.  I don't know why we would want false.

This is not a request for a change.

Reply:
Eventually, we need to implement `OnCheckForUpdates` using this method. But strictly speaking - it is not that trivial, as we may want to add some "loader" animation for the time request is performed. Like spinning circle or so. In this case - we will need to inform users about the errors